app/workers/deletes.php

```diff
@@ -369,14 +369,6 @@ protected function deleteProject(Document $document): void
         // Delete project certificates
```

Comment:
can. you remove this comment now that we don't do this line this line anymore

Reply:
done